setup_wizard/material_import_setup/game_material_importers.py

```diff
@@ -121,19 +114,37 @@ def __init__(self, blender_operator, context):
         )
 
     def import_materials(self):
-        return super().import_materials()  # Genshin Impact Material Importer
+        status = super().import_materials()  # Genshin Impact Material Importer
+
+        if status == {'FINISHED'}:
+            return status
```

Comment:
Technically could just return `{'FINISHED'}` so you don't have to think about the status being returned